Assets/Scripts/Door.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Door : MonoBehaviour {
+
+    [SerializeField]
+    private string sceneToLoad;
+
+    private bool isPlayerInTrigger;
+
+    // Bug: Triggers door twice due to double colliders on Player
+    //private void OnTriggerStay2D(Collider2D collision)
+    //{
+    //    if (collision.CompareTag("Player"))
+    //    {
+    //        if (Input.GetButtonDown("Activate"))
+    //        {
+    //            Debug.Log("Player activated door");
+    //        }
+    //    }
+    //}
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.CompareTag("Player"))
+        {
+            isPlayerInTrigger = true;
+        }
+    }
+    private void OnTriggerExit2D(Collider2D collision)
+    {
+        if (collision.CompareTag("Player"))
+        {
+            isPlayerInTrigger = false;
+        }
+    }
+
+    private void Update()
+    {
+        if (Input.GetButtonDown("Activate") && isPlayerInTrigger)
+        {
+            Debug.Log("Player activated door");
+            //SceneManager.LoadScene(sceneToLoad);
+        }
+    }
```

Comment:
Maybe take out that commented out code and put it somewhere else like a google doc? It seems like it would be useful for another occassion but it’s not being used in this project and taking up space.